Roommate-Budget-Helper/budget-helper-full-stack-app
# Error Handling and Testing for Login

## Jira Ticket:
RBH-47-testing-for-login-and-registration
RBH-57-Fix-Error-Handling
## Description of Work
This PR was created by David to get login error handling working and updated to fully cover login tests.
## Acceptance Criteria

- Pass all cypress tests
- Properly display errors to the frontend

## How to Test

- read the README.md in /app folder to see how to get the cypress working locally.
- Launch the Xming through the instructions, then run "npm run e2e" in your dev container
- Click the login-and-registration testing option in the cypress menu and let it run.

### List of Changes:

- Register Page
- Login Page
- Register Backend calls
- Login/Register cypress testing

### Screenshots

| Image | Description |
| ----- | ----------- |
|![MicrosoftTeams-image](https://user-images.githubusercontent.com/73263545/228431746-910fd3d9-c74f-40ab-98c7-32e0f77f7c0e.png) | All tests passing on Cypress |
